applications/dashboard/views/user/deleteconfirm.php

```diff
@@ -15,7 +15,8 @@
     else if ($this->Method == 'wipe')
         echo t("All of the user content will be replaced with a message stating the user has been deleted.");
     else
-        echo t("The user content will be completely deleted.");
+        echo t("The user content will be completely deleted."),
+        $this->Form->checkBox('DeleteNotes', t('Delete moderation notes as well'), ['value' => '1']);
```

Comment:
Can we do `DeleteModerationInfo` as the field and make it "Delete moderation information." for the label.